### ISSUE TITLE
App get stucked on "create admin account"

### ISSUE THREAD

**Issue by skorakora** [jellyfin/jellyfin-android#880]
### Describe the bug

Every single lauch of the app ends up like this.

When I am connecting to host tough app it asks to set admin password, field for password is filed with dots.
Admin account was created before, so I have no idea why app still asks for admin account creation.
Even If I woud like to create admin account trough app I am unable because app auto-fills password field with "dots" and when I leave everything empty it says "Password, and password confirmation must match"

This only appears when I am accessing server trough android app.

When I first installed the app it sucessfully connected to server, but when I deleted the app this error started to show up.

### Application version

2.3.0

### Where did you install the app from?

Google Play

### Device information

RedminNote 9 pro

### Android version

10 QKQ1

### Jellyfin server version

10.7.6

### Which video player implementations does this bug apply to?

- [ ] Web player (default)
- [ ] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

**Comment by regeter** [comment]
I have exactly this same issue.
The weird thing is that this used to work.

**Comment by regeter** [comment]
I was able to work around it. I saw in the logs that it tried to create my existing user.
So I changed it to a different username and it actually created a NEW admin user.

I believe there are two root causes:
1.) Android doesn't detect that an admin user was already created using the web.
2.) Using the web, I added my local LAN as "local" which I dearly hope enabled this "admin account" creation. Keep in mind that Admin users can delete files...

**Comment by regeter** [comment]
I was able to work around it. I saw in the logs that it tried to create my existing user and failed since that user already existed.
So I changed it to a different username and it actually created a NEW admin user.

I believe there are two root causes:
1.) Android doesn't detect that an admin user was already created using the web.
2.) Using the web, I added my local LAN as "local" which I believe enabled this "admin account" creation.
